wordCount.rb

```diff
@@ -0,0 +1,33 @@
+class WordCounter
+  attr_accessor :file, :input, :output
+  def initialize(file)
+    @file = file
+    @input = ""
+    @output = {}
+  end
+
+  def parse
+    File.foreach(@file) {|x| count(x)}
+  end
+
+  def print
+    @output = @output.sort_by {|key, value| value}
+    @output.each {|key, value| puts "#{value} - #{key}"}
+  end
+
+  private
+  def count(input)
+    input = input.split
+    input.each do |word|
+      if @output[word] == nil
```

Comment:
If the default hash is set to 0, this method can just be``input.split.each{ |word| @output[word] += 1 }`` because you don't need to handle the initialization case.